src/components/PopularRestaurants/PopularRestaurants.tsx

```diff
@@ -56,7 +56,7 @@ const PopularRestaurants = () => {
             src={AllResturantsIcon}
             alt="AllResturantsIcon"
             onClick={() => {
-              console.log("button clicked");
```

Comment:
why do you need it?

Reply:
for future uses if the button will clicked, if I will need to implement this page, same for all the buttons